lib/appsignal/error_handler.ex

```diff
@@ -120,36 +120,28 @@ defmodule Appsignal.ErrorHandler do
   defp match_error_format('Error in process ' ++ _, [pid, {reason, stack}]) do
     msg = "Process #{inspect pid} raised an exception"
     {reason, msg} = extract_reason_and_message(reason, msg)
-    {pid, reason, msg, format_stack(stack), nil}
+    {pid, reason, msg, Backtrace.from_stacktrace(stack), nil}
   end
 
   defp match_error_format('** Generic server ' ++ _, [pid, _last, _state, reason]) do
     {reason, stack} = maybe_extract_stack(reason)
     {reason, msg} = extract_reason_and_message(reason, "GenServer #{inspect pid} terminating")
-    {pid, reason, msg, format_stack(stack), nil}
+    {pid, reason, msg, Backtrace.from_stacktrace(stack), nil}
   end
 
   defp match_error_format('** Task ' ++ _, [pid, starter, function, args, reason]) do
     {reason, stack} = maybe_extract_stack(reason)
     msg = "Task #{inspect pid} started from #{inspect starter} terminating. Function: #{inspect function}, args: #{inspect args}"
     {reason, msg} = extract_reason_and_message(reason, msg)
-    {pid, reason, msg, format_stack(stack), nil}
+    {pid, reason, msg, Backtrace.from_stacktrace(stack), nil}
   end
 
   # FIXME add test coverage for this one
   defp match_error_format('Ranch listener ' ++ _, [_, _, pid, {{reason, stack}, initial}]) do
     conn = extract_conn(initial)
     msg = "HTTP request #{inspect pid} crashed"
     {reason, msg} = extract_reason_and_message(reason, msg)
-    {pid, reason, msg, format_stack(stack), conn}
-  end
-
-  # Format the stack trace as an array of strings
-  @doc false
-  def format_stack(stacktrace) do
```

Comment:
is this used by any user's themselves? i.e. should we deprecate the function before removing it?